packages/shared/routes/dashboard/wallet/Wallet.svelte

```diff
@@ -1,47 +1,37 @@
 <script lang="typescript">
     import { isDeepLinkRequestActive } from '@common/deep-links'
-    import { accountRoute } from '@core/router'
-    import { AccountRoute } from '@core/router/enums'
-    import { AccountActionsModal, DashboardPane, Text, Modal } from 'shared/components'
+    import { AccountSummaryAndAssetsPane, AccountHistoryPane, LineChartPane, BarChartPane } from 'shared/components'
     import { loggedIn, sendParams } from 'shared/lib/app'
     import { localize } from '@core/i18n'
-    import { displayNotificationForLedgerProfile, promptUserToConnectLedger } from 'shared/lib/ledger'
+    import { displayNotificationForLedgerProfile } from 'shared/lib/ledger'
     import { addProfileCurrencyPriceData } from 'shared/lib/market'
     import { showAppNotification } from 'shared/lib/notifications'
-    import { closePopup, openPopup } from 'shared/lib/popup'
+    import { openPopup } from 'shared/lib/popup'
     import {
         activeProfile,
         isLedgerProfile,
         isSoftwareProfile,
         isStrongholdLocked,
         setMissingProfileType,
     } from 'shared/lib/profile'
-    import { AccountIdentifier } from 'shared/lib/typings/account'
     import { LedgerErrorType } from 'shared/lib/typings/events'
     import {
-        profileManager,
         api,
         asyncSyncAccounts,
-        getAccountMessages,
-        getStardustAccount,
         getSyncAccountOptions,
         hasGeneratedALedgerReceiveAddress,
         isFirstSessionSync,
-        prepareAccountInfo,
         removeEventListeners,
         selectedAccount,
         selectedAccountId,
-        updateBalanceOverview,
         wallet,
     } from 'shared/lib/wallet'
     import { initialiseListeners } from 'shared/lib/walletApiListeners'
     import { onMount } from 'svelte'
-    import { AccountAssets, AccountBalance, AccountHistory, BarChart, LineChart, ManageAccount, Send } from './views/'
 
-    const { accounts, accountsLoaded, internalTransfersInProgress } = $wallet
-
-    let modal: Modal
+    const { accounts, accountsLoaded } = $wallet
 
+    // TODO: move to dashboard or lib
```

Comment:
There's quite a lot of TODO comments in the code already, so we probably shouldn't add anymore. We can make a task for later?

Reply:
Yep I will leave the deep comment in and make a task too. Lots of todos are already contained with in larger tasks on backlog. And stardust develop is very much a work in progress branch until we finish a lot of refactoring.